articles/urls.py

```diff
@@ -0,0 +1,22 @@
+from django.urls import path
+
+from articles.views import ArticleDetailView, ArticleListView
+
+urlpatterns = [
+    path("articles/", ArticleListView.as_view(), name="articles"),
+    path(
```

Comment:
ArticleDetailView paths to be just 1, the request method is the one that will differentiate which mmethod will be called

Reply:
done